packages/js/experimental/tsconfig-cjs.json

```diff
@@ -2,6 +2,7 @@
 	"extends": "../tsconfig-cjs",
 	"compilerOptions": {
 		"outDir": "build",
+		"noCheck": true,
```

Comment:
Why was this added @rjchow ?

Reply:
I've answered myself. ` npx tsc` didn't show me errors, but I managed to see them in the end at build time.